### PR TITLE
Feature/CATX A1 ODT@29

### PR DESCRIPTION
**Pull request by JaneMcBrain**
https://jira.catena-x.net/browse/A1ODT-29
Also it implements already nodes and connections

### CODE REVIEW

File: devops-tooling/dashboard/src/App.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.app{
```
Comment (JaneMcBrain) [inline devops-tooling/dashboard/src/App.scss:1]: @dhiraj-wakchaure I know that react starts class names with capital letters. For me personally I prefer lowercase. Anyway I totally understand the benefits of having a component class with capital and functional classes like 'space-bottom-10' all lowercase. What do you think?

File: devops-tooling/dashboard/src/App.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+
+  &__header{
```
Comment (JaneMcBrain) [inline devops-tooling/dashboard/src/App.scss:6]: I also like the BEM methodology for better structuring. What do you think?

File: devops-tooling/dashboard/src/App.tsx [hunk 3]
```diff
@@ -4,9 +4,12 @@ import Dashboard from './components/Dashboard/Dashboard';
 
 function App() {
   return (
-    <div>
+    <div className="app">
+      <header className="app__header">
```
Comment (JaneMcBrain) [inline devops-tooling/dashboard/src/App.tsx:8]: This should be for sure it's own 'Header' component. Maybe we can have a layout folder for those kind of components?

File: devops-tooling/dashboard/src/App.scss (hunk 1 above)
Comment (dhiraj-wakchaure) [reply devops-tooling/dashboard/src/App.scss:1]: lower case sounds better to me also , we can keep this

File: devops-tooling/dashboard/src/components/Dashboard/Dashboard.scss [hunk 4]
```diff
@@ -1,2 +1,4 @@
-.Dashboard {
-}
\ No newline at end of file
+.dashboard {
+  height: 100%;
+  background-color: #777;
```
Comment (JaneMcBrain) [inline devops-tooling/dashboard/src/components/Dashboard/Dashboard.scss:3]: @dhiraj-wakchaure I think in the future we will have certain colors from the Catena-X styleguide. I would add these colors to a "variables.scss" file and reference those to a functional class like 'bg-primary' or 'color-primary' wha do you think?

File: devops-tooling/dashboard/src/App.scss (hunk 2 above)
Comment (dhiraj-wakchaure) [reply devops-tooling/dashboard/src/App.scss:6]: I don't have enough knowledge to have opinion on this , we can keep in this way.

File: devops-tooling/dashboard/src/App.tsx (hunk 3 above)
Comment (dhiraj-wakchaure) [reply devops-tooling/dashboard/src/App.tsx:8]: yeah , may be on next story we can create common layout component and try to organize

File: devops-tooling/dashboard/src/components/Dashboard/Dashboard.scss (hunk 4 above)
Comment (dhiraj-wakchaure) [reply devops-tooling/dashboard/src/components/Dashboard/Dashboard.scss:3]: yeah  I think so we can keep in this way , and in next story we might pickup any library that would support this config

File: devops-tooling/dashboard/src/App.scss (hunk 2 above)
Comment (JaneMcBrain) [reply devops-tooling/dashboard/src/App.scss:6]: https://en.bem.info/methodology/

File: devops-tooling/dashboard/src/components/Dashboard/Dashboard.scss (hunk 4 above)
Comment (JaneMcBrain) [reply devops-tooling/dashboard/src/components/Dashboard/Dashboard.scss:3]: What do you mean with library? Actually we don't need a lib for that. I will prepare an example.

**Review by dhiraj-wakchaure — APPROVED** [review]

**Pull request merged**